ACE/ace/config-integrity-common.h

```diff
@@ -0,0 +1,166 @@
+#ifndef ACE_CONFIG_INTEGRITY_COMMON_H
+#define ACE_CONFIG_INTEGRITY_COMMON_H
+
+#define ACE_INTEGRITY
+
+#define ACE_LACKS_GETPGID
+#define ACE_LACKS_SETPGID
+#define ACE_LACKS_SETREUID
+#define ACE_LACKS_SETREGID
+#define ACE_LACKS_SETSID
+#define ACE_LACKS_GETEGID
+#define ACE_LACKS_SETEGID
+#define ACE_LACKS_GETUID
+#define ACE_LACKS_SETUID
+#define ACE_LACKS_GETEUID
+#define ACE_LACKS_SETEUID
+#define ACE_LACKS_GETGID
+
+#define _REENTRANT
+
+#define ACE_MT_SAFE 1
+#define ACE_HAS_THREADS
+#define ACE_MAX_NUM_THREADS 64
+
+#define ACE_HAS_STRERROR
+#define ACE_HAS_SIG_ATOMIC_T
+
+#define ACE_HAS_SIGINFO_T
+#define ACE_LACKS_SIGINFO_H
+#define ACE_LACKS_UCONTEXT_H
+#define ACE_HAS_SIG_C_FUNC
+#define ACE_LACKS_SI_ADDR
+
+#ifndef ACE_LACKS_POSIX
+# define ACE_HAS_POSIX_NONBLOCK
+#endif
+#define ACE_HAS_DIRENT
+
+#define ACE_PAGE_SIZE 4096
+
+#define ACE_LACKS_KEY_T
+
+#define ACE_LACKS_ISCTYPE
+#define ACE_LACKS_WCSICMP
+#define ACE_LACKS_WCSNICMP
+#define ACE_LACKS_WCSDUP
+#define ACE_LACKS_WCSTOK
+#define ACE_LACKS_ALPHASORT
+#define ACE_LACKS_ITOW
+
+#define ACE_HAS_SIGWAIT
+#define ACE_HAS_PTHREAD_SIGMASK_PROTO
+
+#define ACE_LACKS_SUSECONDS_T
+#define ACE_LACKS_USECONDS_T
+#define ACE_HAS_POSIX_TIME
+#define ACE_HAS_STRDUP_EMULATION
```

Comment:
Is ACE_HAS_STRDUP_EMULATION really required for integrity, seems the last platform still using this, would be nice when we could remove this from ACE

Reply:
I don't think Integrity has `strdup` as far as I can tell.  It's also not in std C until C23 so there's bound to be some non-POSIX platform that needs it.